ui/perf_counter_model.cpp

```diff
@@ -18,6 +18,14 @@
 #include <QDebug>
 
 #include "dive_core/available_metrics.h"
+#include "dive_core/perf_metrics_data.h"
+
+enum class FixedHeader : uint8_t
```

Comment:
you could probably avoid the static_cast<int> here and still keep the scope by doing something like

struct FixedHeader 
{
enum Type : int
{
    kDrawID,
    kLRZState,
    kFixedHeaderCount
};
};